routes/error-tracker.js

```diff
@@ -191,6 +195,7 @@ function getHandler(req, res, next) {
       res.end();
       winston.error(appEngineProjectId, 'Cannot write to Google Cloud Logging: '
         + url.parse(req.url, true).query['v'], err);
+      return;
```

Comment:
I think if I don't return here, there may be an attempt to send a response below which causes the rejected promise error since it tries to set headers of a response that was set in this branch.